dockers/docker-base/Dockerfile.j2

```diff
@@ -29,6 +29,7 @@ RUN apt-get -y install \
     vim-tiny           \
     perl               \
     python             \
+    psmisc             \
```

Comment:
why do we need this psmisc in the docker-base?

Reply:
@pavel-shirshov , I believe sonic-utilties has psmisc dependency. 

anyway, @mkbalani  please remove this from this PR. If it is an issue, please raise separate PR for this.

Reply:
removed for now